cgt_calc/model.py

```diff
@@ -130,6 +130,7 @@ class BrokerTransaction:
     amount: Decimal | None
     currency: str
     broker: str
+    isin: str | None
```

Comment:
let's avoid having to modify all constructors, just the ones that need it
```suggestion
    isin: str | None = None
```